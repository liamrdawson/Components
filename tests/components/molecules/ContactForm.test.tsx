import { jsx } from '@emotion/react'
import {render} from '@testing-library/react'
import { ContactForm } from '../../../src'

test('Snapshot Test: Contact Form', () => {
  const {container} = render(
    <ContactForm>Heading Level One</ContactForm>,
  )
  expect(container).toMatchSnapshot()
})
