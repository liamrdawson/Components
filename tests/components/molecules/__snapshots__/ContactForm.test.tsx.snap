// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test: Contact Form 1`] = `
<div>
  <form
    class="css-154v6jj-ContactForm"
  >
    <div
      class="css-6n93mb-TextInput"
    >
      <label
        class="css-2x82w0-TextInput"
        for="First Name"
      >
        First Name
      </label>
      <input
        class="css-18ek46i-TextInput"
        id="First Name"
        name="First Name"
        type="text"
      />
    </div>
    <div
      class="css-1n3bh1d-TextInput"
    >
      <label
        class="css-2x82w0-TextInput"
        for="Last Name"
      >
        Last Name
      </label>
      <input
        class="css-18ek46i-TextInput"
        id="Last Name"
        name="Last Name"
        type="text"
      />
    </div>
    <div
      class="css-boj1sm-TextInput"
    >
      <label
        class="css-2x82w0-TextInput"
        for="Email Address"
      >
        Email Address
      </label>
      <input
        class="css-18ek46i-TextInput"
        id="Email Address"
        name="Email Address"
        type="email"
      />
    </div>
    <div
      class="css-1agg0r2-TextInput"
    >
      <label
        class="css-2x82w0-TextInput"
        for="Subject"
      >
        Subject
      </label>
      <input
        class="css-18ek46i-TextInput"
        id="Subject"
        name="Subject"
        type="text"
      />
    </div>
    <div
      class="css-12anhju-TextArea"
    >
      <label
        class="css-1bawu2g-TextArea"
        for="Message"
      >
        Message
      </label>
      <textarea
        class="css-1jf3zmb-TextArea"
        id="Message"
        name="Message"
      />
    </div>
    <button
      class="css-1195o8y-PrimaryButton"
      type="button"
    >
      Send
    </button>
  </form>
</div>
`;
